doc/bridge_packages/flame_forge2d/joints.md

```diff
@@ -112,3 +112,51 @@ to 0, the higher the value, the less springy the joint becomes.
 ```{warning}
 Do not use a zero or short length.
 ```
+
+
+### `FrictionJoint`
+
+A `FrictionJoint` is used for simulating friction in a top-down game. It provides 2D translational
```

Comment:
It didn't have to be a top-down game, right?

Reply:
It certainly works with non-zero gravity, but it was designed specifically for top-down games. From [Box2D documentation](https://box2d.org/documentation/md__d_1__git_hub_box2d_docs_dynamics.html#autotoc_md94):

> The friction joint is used for top-down friction

I also confirmed it in the Box2D discord:

> Erin — 03/02/2023 11:12 PM
The friction joint is designed for a top down game. Imagine you have a top down game where a box is sliding across the floor (which is the plane of the screen). This joint applies friction between the box and the floor. Now suppose you have a top down game where the box is on the floor of a moving vehicle. The floor is now moving, so you need a second body. The friction joint is not related the friction that you get when shapes collide in the x-y space of the screen plane, instead it is related to friction across the z direction, perpendicular to the screen.